cogeo_mosaic/mosaic.py

```diff
@@ -171,6 +189,18 @@ def _create_mosaic(
             "version": "1.0.0",
         }
 
+        mosaic_003 = {
+            "tilematrixset": tilematrixset,
```

Comment:
We only add `tilematrixset` if the user provided it, if not the application should assume `WebMercatorQuad` was used